packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -0,0 +1,140 @@
+          targetDistro: 'debian',
+        }),
+      ).toBe('debian-openssl-1.1.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+        prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
+        Please manually install OpenSSL via \`apt-get update -y && apt-get install -y openssl\` and try installing Prisma again. If you're running Prisma on Docker, you may also try to replace your base image with \`node:lts-slim\`, which already ships with OpenSSL installed.
+      `)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+
+    it('arch (arch), amd64 (x86_64), openssl-undefined', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: undefined,
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: 'arch',
+          originalDistro: 'arch',
+          targetDistro: 'debian',
+        }),
+      ).toBe('debian-openssl-1.1.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+
+    it('unknown (unknown), amd64 (x86_64), openssl-3.0.x', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: '3.0.x',
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: undefined,
+          originalDistro: 'unknown',
+          targetDistro: undefined,
+        }),
+      ).toBe('debian-openssl-3.0.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+        prisma:warn Prisma doesn't know which engines to download for the Linux distro "unknown". Falling back to Prisma engines built "debian".
+        Please report your experience by creating an issue at https://github.com/prisma/prisma/issues so we can add your distro to the list of known supported distros.
+      `)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+
+    it('unknown (unknown), amd64 (x86_64), openssl-undefined', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: undefined,
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: undefined,
+          originalDistro: 'unknown',
+          targetDistro: undefined,
+        }),
+      ).toBe('debian-openssl-1.1.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
```

Comment:
Warn should also say " prisma:warn Prisma doesn't know which engines to download for the Linux distro "unknown". Falling back to Prisma engines built "debian"." here?

Reply:
That's possible, let me double-check that